packages/@coorpacademy-components/src/molecule/skills-chart-side-information-item/test/fixtures/default-courses-completed.js

```diff
@@ -0,0 +1,11 @@
+import Icon from '../../../../atom/icon/test/fixtures/default-with-only-icon-name-and-bg-color-green-hex';
+
+const iconProps = Icon.props;
+export default {
+  props: {
+    title: 'Courses completed',
+    value: '165',
+    legend: 'on <b>focused skills</b>',
+    icon: {...iconProps, preset: 's'}
```

Comment:
j'ai pas compris pourquoi tu prends Iconprops?
C'est Icon qui doit gérer le default.

Ceci devrait faire l'affaire.
```suggestion
    icon: {preset: 's'}
```

(idem pour les autres fixtures)

Reply:
parce que si tu veux avoir une icon, il te faut au moins le nom de l'icon 
il te faut au moins faire : 
`icon: {iconName: book-open-cover} `
en plus pour visualiser les icons dont j'ai besoin, il faudrait spécifier le background, mais le size aussi, sinon j'aurai un preset "m" par défault
alors au lieux de faire ca: 
`icon: {iconName: book-open-cover, backgroundColor: '#D9F4F7', preset: 's'}`
je fais ca: 
`icon: {...iconProps, preset: 's'}`
puisque l'icon que je veux visualiser a dèja une fixture (dans ce cas default-with-only-icon-name-and-bg-color-green-hex)